funnel/templates/space.html

```diff
@@ -125,9 +125,16 @@
   </strong>
 </p>
 {%- if space.sessions %}
-  <p>
-    <a href="{{ space.url_for('schedule') }}" class='btn btn-primary btn-sm'>{% trans %}View Schedule{% endtrans %}</a>
-  </p>
+  <ul class="list-inline">
+    <li>
+      <a href="{{ space.url_for('schedule') }}" class='btn btn-primary btn-sm'>{% trans %}View Schedule{% endtrans %}</a>
```

Comment:
There needs to be an if condition for this too. A new proposal space has no schedule. However, this requires a separate issue.